### ISSUE TITLE
Fails on node 5

### ISSUE THREAD

**Issue by apsavin** [arikon/bower-npm-install#17]
It seems that bower-npm-install fails on node 5 (because of npm 3, I suppose):

```
$ node ./node_modules/.bin/bower-npm-install --non-interactive
(node) util.error is deprecated. Use console.error instead.
Error: Cannot find module 'bower/node_modules/bower-logger'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/home/travis/build/apsavin/bnsf/node_modules/bower-npm-install/lib/install.js:9:14)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
```

**Comment by arikon** [comment]
@apsavin Could you make a PR?

**Comment by arikon** [comment]
Landed in `bower-npm-install@0.5.10`
